src/sentry/plugins/sentry_mail/models.py

```diff
@@ -193,6 +195,29 @@ def notify(self, notification):
             context=context,
         )
 
+    def notify_digest(self, project, digest):
```

Comment:
I figure this will get cleaned up as we figure out what we actually want this to do.